docs/editor/accessibility.md

```diff
@@ -158,7 +158,9 @@ Certain features such as folding and minimap (code overview) are disabled when i
 
 ## Terminal accessibility
 
-Output in the Integrated Terminal can be navigated through by using the "navigation mode" commands available in the Command Palette (press `kbstyle(F1)` and search for "terminal navigation mode").
+Terminal accessibility help can be toggled via `kbstyle(Alt+F1)` which describes some useful tips when using a screen reader. One of the tips described is to use `kbstyle(Shift+Tab)` to access the terminal's buffer, this will enter the screen reader's browse mode automatically (depending on your screen reader) and provide an accessible view of the entire terminal's buffer.
```

Comment:
Prefer to use `kb(workbench.action.terminal.showAccessibilityHelp)` than hard-coding the keyboard shortcut so if there are platform-specific keybindings, users will see the appropriate keybindings.
In this case, all platforms are now "Alt+F1" but earlier in milestone Windows had been "Shift+F1"

Reply:
Similar for `kb(workbench.action.terminal.enterAccessibilityMode)` even though now all platforms use "Shift+Tab" it could change in the future

Reply:
It's alt+f1 and shouldn't change, but yeah that's better :+1: